compliance/azure/instances_without_fnm_agent/Azure_Instances_not_running_FlexNet_Inventory_Agent.pt

```diff
@@ -0,0 +1,194 @@
+name "Azure Instances not running FlexNet Inventory Agent"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on any instances that are not running the FlexNet Inventory Agent. \n See the [README](https://github.com/flexera/policy_templates/tree/master/compliance/azure/instances_without_fnm_agent) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
I suggest the following:
Checks Azure instances that are not running the FlexNet Inventory Agent

Reply:
Changed, thanks.

Reply:
Sorry, I must have tagged the wrong line.  The short description should change to: Check Azure instances that are not running the FlexNet Inventory Agent

Reply:
Azure Instances not running FlexNet Inventory Agent